README.md

```diff
@@ -5,30 +5,31 @@ One of the ways the MI team evaluates a potential Ridepooling service is by simu
+The Simulator itself is ready to go. Your mission is to integrate it in a solution that enables users to trigger a simulation and save part of the KPIs (booking distance bins) in database and see the simulation results visualized in some way accessible through a webpage.
 
 The code should not only fulfill the functional requirements but also be production-ready: that means clean, maintanable, well-documented, well-tested code with any necessary instructions to build/run it.
 
 
 ### Requirements
 - A webpage where the user can trigger a simulation and see its results visualized.
+- Run the load data file `load_data.py` and read the bounding box of Berlin `de_berlin` from sqlite to be used as simulation input
 - An API where we can trigger the simulation and get the results back.
-- All the results of the simulation visualized. It's up to you to choose how to visualize the data and justify your choices.
+- Save the results of the booking distance bins in the `booking_distance` table in sqlite
+- Visualize the most popular pickups and dropoffs from the results of the simulation. It's up to you to choose how to visualize the data and justify your choices.
 
 The documentation for the Simulator can be found in the `simulator` directory.
 
 
 ### Out of scope
 - **User authentication/login.** You don't have to implement login, API keys, etc. Assume that if a user has access, they should.
 - **Any modifications to the Simulator.** Assume that it works as expected and outputs only what's necessary. Of course, if you would like to adjust the formatting of the payload for example to optimize/better match your visualization, you can do so. But please justify any changes accordingly.
-- **Customized input to trigger the simulation(s).** It's okay if the input payload is hardcoded or randomized. A simple button to trigger the simulation via the front-end is enough. It is expected, though, that any mocking would be done in the front-end layer.
-- **Storing the simulation results in some sort of database.**
-- **Testing the functionality of the Simulator itself.**
+- **Customized input to trigger the simulation(s).** It's okay if the number of request input is hardcoded or randomized. A simple button to trigger the simulation via the front-end is enough. It is expected, though, that any mocking would be done in the front-end layer.
+- **Testing the functionality of the Simulator.**
 
 
 ### Technical assumptions
-- The API/wrapper around the Simulator module should be built with Python3.
-- The front-end/webpage should be built with [React](https://reactjs.org/).
+- The API/wrapper around the Simulator module should be built with Python3 or preferred programming language.
+- The front-end/webpage should be built with [React](https://reactjs.org/) or any javascript alternative.
```

Comment:
These two lines are quite confusing. I'm not sure if it is preferred to use Python3 and React or pick whatever language and front-end technology I want.

What do you think about listing the languages we will accept the code challenge, or say that the candidate can choose whatever they want without mentioning Python3 or React?

Reply:
Then we can only mention that we preferred to be done in Python 3 and React, what do you think?

Reply:
Makes sense to me.